applications/FluidDynamicsApplication/python_scripts/navier_stokes_compressible_solver.py

```diff
@@ -197,24 +148,35 @@ def _ExecuteAfterReading(self):
 
         check_and_prepare_model_process_fluid.CheckAndPrepareModelProcess(self.main_model_part, prepare_model_part_settings).Execute()
 
-
-    #def _SetCompressibleElementReplaceSettings(self):
-        #domain_size = self.main_model_part.ProcessInfo[KratosMultiphysics.DOMAIN_SIZE]
-        #self.settings.AddEmptyValue("element_replace_settings")
-
-        #if(domain_size == 3):
-            #self.settings["element_replace_settings"] = KratosMultiphysics.Parameters("""
-            #{
-                #"element_name":"CompressibleNavierStokes3D4N",
-                #"condition_name": "SurfaceCondition3D3N"
-            #}
-            #""")
-        #elif(domain_size == 2):
-            #self.settings["element_replace_settings"] = KratosMultiphysics.Parameters("""
-            #{
-                #"element_name":"CompressibleNavierStokes2D3N",
-                #"condition_name": "LineCondition2D2N"
-            #}
-            #""")
-        #else:
-            #raise Exception("Domain size is not 2 or 3!!")
+    def _CreateScheme(self):
```

Comment:
this seems to be again have much copy-pasted code from the baseclass

Reply:
This solver is legacy (I'm pretty sure no one is using it).

I did it in that way to avoid changing the default settings.